Block/Info/Cc.php

```diff
@@ -31,15 +31,8 @@ public function getCcTypeName()
 
         if (isset($types[$ccType])) {
             return $types[$ccType]['name'];
-        }
-        // TODO::Refactor this block after tokenization of the alternative payment methods.
-        // This elseif block should be removed after the tokenization of the alternative payment methods (In progress: PW-6764). More general approach is required.
-        // Also remove `sepadirectdebit` from translation files.
-        elseif ($ccType == 'sepadirectdebit') {
-            return __('sepadirectdebit');
-        }
-        else {
-            return __('Unknown');
+        } else {
+            return $ccType;
```

Comment:
How about adding them to [this](https://github.com/Adyen/adyen-magento2/blob/develop/etc/adyen_payment.xml) instead of showing directly to the shopper?

Reply:
yes I added all the variants :) @candemiralp